### PR TITLE
APEXMALHAR-2399 default constructor was directly throwing an exception

### PR DESCRIPTION
**Pull request by KapoorHitesh**
@chinmaykolhatkar please review

### CODE REVIEW

File: library/src/test/java/org/apache/apex/malhar/lib/window/impl/PojoInnerJoinTestApplication.java [hunk 1]
```diff
@@ -0,0 +1,415 @@
...
+    dag.addStream("SalesToJoin", salesGenerator.outputsales, op.input);
+    dag.addStream("ProductToJoin", productGenerator.outputproduct, op.input2);
+
+    dag.addStream("results", op.output, results.input);
+    dag.addStream("wm1", salesGenerator.watermarkDefaultOutputPort,op.controlInput);
+    dag.addStream("wm2", productGenerator.watermarkDefaultOutputPort,op.controlInput2);
+
+  }
+
+  public static void main(String[] args) throws Exception
```
Comment (chinmaykolhatkar) [inline library/src/test/java/org/apache/apex/malhar/lib/window/impl/PojoInnerJoinTestApplication.java]: Can you move this to test method instead of main?

**Review by chinmaykolhatkar — CHANGES_REQUESTED** [review]
One minor change. Else looks good.

**Comment by KapoorHitesh** [comment]
@chinmaykolhatkar Done the suggested code changes and have rebased it. Please review and merge.

**Comment by chinmaykolhatkar** [comment]
LGTM

**Pull request merged**